pkg/metrics/engine.go

```diff
@@ -102,6 +104,20 @@ func newEngineMetrics() engineMetrics {
 			Name:      "list_objects_duration",
 			Help:      "Accumulated duration of engine list objects operations",
 		})
+
+		containerSize = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Namespace: storageNodeNameSpace,
+			Subsystem: engineSubsystem,
+			Name:      "container_size",
+			Help:      "Accumulated size of all objects in a container",
+		}, []string{containerIDLabelKey})
+
+		payloadSize = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Namespace: storageNodeNameSpace,
+			Subsystem: engineSubsystem,
+			Name:      "payload_size",
```

Comment:
this name seems to generic to me, how about `shard_payload_size/volume`?

Reply:
it will always have a `"shard"` label valued with its ID. do you still think it is required (i dont mind just confirm it, please)?

Reply:
if `shard` will be mentioned, then it's ok